store/src/java/com/zimbra/soap/SoapEngine.java

```diff
@@ -339,8 +339,15 @@ private Element dispatch(String path, Element envelope, Map<String, Object> cont
             if (!doc.getName().equals("BatchRequest")) {
                 doCsrfCheck =  false;
             } else {
-                LOG.info("Only BatchRequest does not have a handler mapped to it. Request: %s, does not have a "
-                    + "handler, log for future handling.", path);
+                StringBuilder sb = new StringBuilder();
+                for (Element req : doc.listElements()) {
+                    if (sb.length() > 0) {
+                        sb.append(",");
+                    }
+                    sb.append(req.getName());
+                }
+                LOG.info("BatchRequest [%s] contains %d sub-request(s) [%s]",
```

Comment:
minor suggestion: change
  "BatchRequest [%s] contains %d sub-request(s) [%s]"
to
  "BatchRequest [%s] contains %d sub-request(s): %s"

This is very minor, but it would seem (to me) to be more consistent with how we currently log other items.